aries-site/src/pages/foundation/voice-and-tone.mdx

```diff
@@ -170,31 +170,54 @@ These concrete rules structure our communications.
 
 ## Capitalization
 
-The general rule is that things that are sentences should follow sentence capitalization rules. And, things that are titles/headings/labels should follow title capitalization rules.
-
-Sentences follow sentence capitalization rules:
-
+There are five types of capitalization. 
+1. Sentence case, where the first letter is capitalized and subsequent words begin with lowercase letters, with some exceptions, such as proper nouns.
+   (example: Storage volume) 
+2. Title case, where most words are capitalized except for "connector" words such as "the," "an," "or," etc.
+   (example: "The Benefits of Cloud Storage Explained")
+3. PascalCase, where there are capital letters inside the word, almost always for branding purposes.
+    (example: GreenLake)
+4. All lower case, where every single word is lower case 
+    (example: "lower case exudes an approachable and casual vibe")
+5. All caps, where every letter is capitalized
+    (example: ALL LETTERS ARE CAPITALIZED)
+
+At HPE, almost all text should be written in sentence case. Never begin a sentence or phrase with a lower case letter because it is too casual for our brand. Do not write in all caps because it is equated with yelling and increases stress.
```

Comment:
```suggestion
At HPE, almost all text should be written in sentence case. Do not write in all caps because it is equated with yelling and increases stress.
```
If you start with a lowercase letter, it's not sentence case -- so can we remove this redundant sentence?